blog.html

```diff
@@ -22,6 +22,106 @@
         <!--#include virtual="includes/_nav.htm" -->
         <div class="right">
             <h1 class="content-title">Blog</h1>
+            <article>
+                <h2 class="bullet">
+                    <a id="apache_kafka_380_release_announcement"></a>
+                    <a href="#apache_kafka_380_release_announcement">Apache Kafka 3.8.0 Release Announcement</a>
+                </h2>
+                26 July 2024 - Josep Prat (<a href="https://twitter.com/jlprat">@jlprat</a>)
+                <p>We are proud to announce the release of Apache Kafka 3.8.0. This release contains many new features and improvements. This blog post will highlight some of the more prominent features. For a full list of changes, be sure to check the <a href="https://downloads.apache.org/kafka/3.8.0/RELEASE_NOTES.html">release notes</a>.</p>
+                <p>See the <a href="https://kafka.apache.org/documentation.html#upgrade_3_8_0">Upgrading to 3.8.0 from any version 0.8.x through 3.7.x</a> section in the documentation for the list of notable changes and detailed upgrade steps.</p>
+                <p>
+                    In a previous release, 3.6,
+                    <a href="https://kafka.apache.org/38/documentation.html#tiered_storage">tiered storage</a> was released as early access feature.
+                    In this release, Tiered Storage now supports clusters configured with multiple log directories (i.e. JBOD feature). This feature still remains as early access.
+                </p>
+                <p>
+                    In the last release, 3.7, <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-858%3A+Handle+JBOD+broker+disk+failure+in+KRaft">KIP-858</a>
+                    was released in early access. Since this version, JBOD in KRaft is no longer considered an early access feature.
+                </p>
+                <p>
+                    Up until now, only the default compression level was used by Apache Kafka. From this version on, a configuration mechanism to specify compression level is included. See <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-390%3A+Support+Compression+Level">KIP-390</a> for more details.
+                </p>
+                <p>
+                    In the last release, 3.7, <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-848%3A+The+Next+Generation+of+the+Consumer+Rebalance+Protocol">KIP-848 The Next Generation of the Consumer Rebalance Protocol</a> was made available as early access. This version includes numerous bug
+                    fixes and the community is encouraged to test and provide feedback. <a href="https://cwiki.apache.org/confluence/display/KAFKA/The+Next+Generation+of+the+Consumer+Rebalance+Protocol+%28KIP-848%29+-+Early+Access+Release+Notes">See the early access release notes for more information.</a>
```

Comment:
Hi @kirktrue, please take a look and let me know if this is what you had in mind. Please note that Apache Kafka hasn't defined yet properly what "early access" means. And we have no concept of "preview" or anything similar yet.

Reply:
Yes. This seems fine. Thank you!